site/docs/tools/deltalake/spark.md

```diff
@@ -98,7 +96,7 @@ Spark support is constantly evolving and the differences in Spark3 vs Spark2.4 i
 
 ### Spark2
```

Comment:
needs to be updated as well?

Reply:
I have removed it, I think this title is no longer needed